### ISSUE TITLE
Prae acid ball deals almost no damage.

### ISSUE THREAD

**Issue by AnuvKH** [cmss13-devs/cmss13#5659]
### Testmerges

#5007, #5392, #5552, #5649, #5628, #5655

### Description of the bug

Acid "shrapnel" from a ball used to do a lot of damage, near-critting people directly on top of it. Now, it seems to do almost zero damage even to those nearby. I think it's a collision issue with the "shrapnel" but its behavior seems way different from before.

### What's the difference with what should have happened?

Acid ball should do way more damage. There was a "fix" to it a few weeks ago that maybe bugged it.

### How do we reproduce this bug?

1. Launch acid ball as prae.
2. Watch it do next to no damage, even for marines directly adjacent to or above.

### Issue Bingo

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by InsaneRed** [comment]
![image](https://github.com/cmss13-devs/cmss13/assets/47158596/a3cfebb2-131c-4c80-ba44-0cae56ac637d)
seems the same, the only difference i noticed is theres a hit reg difference when you click on the mob as opposted to the tile underenath it

**Comment by InsaneRed** [comment]
yeah just seems to be a difference of tile / sprite clicking and alot of rng since its technically shrapnel, so sometimes you get hit by 1 and take 5 damage and sometimes you take up to 108 damage
